### PR TITLE
Associate employee to company

### PR DESCRIPTION
**Pull request by alexandrejuk**

### CODE REVIEW

File: src/database/models/contractor.js [hunk 1]
```diff
@@ -13,11 +13,17 @@ module.exports = (sequelize) => {
...
   contractor.associate = (models) => {
+
     contractor.belongsTo(models.login, {
       scope: {
         type: 'contractor',
       },
     })
+
+    contractor.belongsToMany(models.company, {
+      through: 'contractorCompany',
```
Comment (vitorsilvalima) [inline src/database/models/contractor.js]: through:  models.contractorCompany

Comment (vitorsilvalima) [inline src/database/models/contractor.js]: through: models.contractorCompany

Comment (alexandrejuk) [reply src/database/models/contractor.js]: It is true!

Comment (alexandrejuk) [reply src/database/models/contractor.js]: It is true!

**Pull request merged**